constructor/winexe.py

```diff
@@ -376,11 +376,9 @@ def create(info, verbose=False):
     shutil.copyfile(info['_conda_exe'], join(tmp_dir, '_conda.exe'))
 
     pre_dst = join(tmp_dir, 'pre_install.bat')
-    try:
-        shutil.copy(info['pre_install'], pre_dst)
-    except KeyError:
-        with open(pre_dst, 'w') as fo:
-            fo.write(":: this is an empty pre install .bat script\n")
+    pre_install_script = info.get("pre_install")
```

Comment:
Does that key always exist? Or should we get with a default that equals to False?
```suggestion
    pre_install_script = info.get("pre_install", "")
```